RUFAS/routines/animal/animal_management.py

```diff
@@ -108,10 +108,9 @@ def __init__(self, data, config, feed, weather, time):
         self.cows_culled = []
 
         # list of all the pens on the farm
-        self.all_pens = []
-
+        self.all_pens_ids = []
```

Comment:
This might be nitpicky or just in my head, but it seems to me like `all_pens_ids` is less clear than `all_pen_ids` because the list contains all of the "pen_ids" not all of the "pens_ids"

```suggestion
        self.all_pen_ids = []
```

Reply:
Good point. However, we're migrating away from the old name entirely as the class variable was changed to `all_pens` . In actuality, the variable contains a list of Pen objects and not their IDs, so this change was definitely needed.